### ISSUE TITLE
Update phenotype scripts for v1.6

### ISSUE THREAD

**Issue by empfff** [National-COVID-Cohort-Collaborative/Phenotype_Data_Acquisition#67]
Four new LOINCs added in phenotype v1.6; see Wiki.

**Comment by marshallclark** [comment]
pcornet sql updated

**Comment by LoraLingrey-TriNetX** [comment]
TriNetX sql updated

**Comment by kmkostka** [comment]
Not in OMOP vocabulary yet. Issue [#321](https://github.com/OHDSI/Vocabulary-v5.0/issues/321) raised with OHDSI Vocab team.

**Comment by kmkostka** [comment]
OHDSI has pushed a vocabulary update to include the v1.6 LOINC codes (issue [#321](https://github.com/OHDSI/Vocabulary-v5.0/issues/321) is now resolved). I should be able to update the phenotype over the weekend once we refresh the vocabs in our public Atlas instances. I've raised an Issue [#15](https://github.com/OHDSI/Covid-19/issues/15) with our maintenance team so that these concept sets can be retrieved in a publicly available knowledge base.
